tests/python/tensorrt/test_tensorrt_lenet5.py

```diff
@@ -0,0 +1,103 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import numpy as np
+import mxnet as mx
+from common import *
+from lenet5_common import get_iters
+
+
+def run_inference(sym, arg_params, aux_params, mnist, all_test_labels, batch_size, use_tensorrt):
+    """Run inference with either MXNet or TensorRT"""
+    mx.contrib.tensorrt.set_use_tensorrt(use_tensorrt)
+
+    shared_buffer = merge_dicts(arg_params, aux_params)
+    if not use_tensorrt:
+        shared_buffer = dict([(k, v.as_in_context(mx.gpu(0))) for k, v in shared_buffer.items()])
+
+    executor = sym.simple_bind(ctx=mx.gpu(0),
```

Comment:
This is still using shared_buffer to pass in weights.

Per our last discussion could you make a new API. For example, mx.sym.contrib.tensorrt_simple_bind.

Reply:
I overlooked this part. Thank @piiswrong for pointing it out. @KellenSunderland I believe we all agreed that symbol binding API for TRT should be also in the contrib. Could you make the change? Thanks.

Reply:
Yeah I've got a version of an implementation that does it.  Shouldn't take me too long to add it.  I wasn't 100% sure this was what was being asked for offline.